homeassistant/components/tradfri/sensor.py

```diff
@@ -17,6 +22,14 @@
 from .coordinator import TradfriDeviceDataUpdateCoordinator
 
 
+@dataclass
+class TradfriSensorEntityDescription(SensorEntityDescription):
+    """Class describing Tradfri sensor entities."""
+
+    should_set_unique_id: bool = False
+    value: Callable[[TradfriSensor], Any | None] = None  # type: ignore
```

Comment:
How would I type this correctly?
```
homeassistant/components/tradfri/sensor.py:30: error: Incompatible types in assignment (expression has type "None", variable has type "Callable[[TradfriSensor], Optional[Any]]")  [assignment]
Found 1 error in 1 file (checked 1 source file)
```

Reply:
If it's optional.
```suggestion
    value: Callable[[TradfriSensor], Any | None] | None = None
```

Reply:
If the attribute is not optional we need to define a mixin dataclass with required attributes and inherit from this mixin in our custom entity description class. Search for mixin in other sensor platforms for examples.

Reply:
Setting value is not optional but the lambda will return either an `int` or (if the fan is turned off): `None`,